macros/LearnSidebar.ejs

```diff
@@ -711,29 +711,29 @@ var text = mdn.localStringMap({
       'Design_and_accessibility': '設計與親合度',
     'How_to_contribute': '如何貢獻'
   },
-  'fr-FR': {
-    'Complete_beginners': 'Les débutants commence ici!',
+  'fr': {
+    'Complete_beginners': 'Bienvenue aux débutants!',
     'Getting_started_with_the_web': 'Commencer avec le Web',
       'Getting_started_with_the_web_overview': 'Vue d\'ensemble de Commencer avec le Web',
       'Installing_basic_software': 'Installation des outils de base',
       'What_will_your_website_look_like': 'Quel sera l\'aspect de votre site web ?',
       'Dealing_with_files': 'Gérer les fichiers',
-      'HMLT_basics': 'Les bases du HTML',
+      'HTML_basics': 'Les bases de HTML',
       'CSS_basics': 'Les bases de CSS',
       'JavaScript_basics': 'Les bases de JavaScript',
       'Publishing_your_website': 'Publier votre site web',
       'How_the_Web_works': 'Le fonctionnement du Web',
      'HTML_structuring_the_Web': 'HTML',
-      'Introdctiuon_to_HTML': 'Introduction au HTML',
+      'Introduction_to_HTML': 'Vue d'ensemble de Introduction au HTML',
```

Comment:
I think the "Vue d'ensemble" should be used for the `Introduction_to_HTML_overview` slug (and not for the `Introduction_to_HTML` one). Also, be careful to escape quotes in "d'ensemble" since the JSON is now invalid.